src/Server/Remote/PhpSecLib.php

```diff
@@ -102,10 +104,15 @@ public function checkConnection()
     /**
      * {@inheritdoc}
      */
-    public function run($command)
+    public function run($command, $timeout = null)
     {
+        if ($timeout === null) {
+            $timeout = self::TIMEOUT;
+        }
+
         $this->checkConnection();
 
+        $this->sftp->setTimeout($timeout);
         $result = $this->sftp->exec($command);
```

Comment:
@elfet it is related to my latest temp.fixed issue with timeout(see PR #429 )
Currenlty if timeout will be reached deployer will silently stop task without any notification and will continue with the next one. This exception will at least help to understand what was happen. 

This flag will be set to true if timeout happen:
https://github.com/phpseclib/phpseclib/blob/7da4f2e6307d11c5984906dafe3c53d4f7c157a1/phpseclib/Net/SSH2.php#L2823

Here is oficial documentation about timeout: http://phpseclib.sourceforge.net/ssh/examples.html#timeout